File: 2.POO/Resto/Inventario/Primera version/C++/Inventario.cpp
#include<iostream>
#include<cstdlib>
using namespace std;
#include"Inventario.h"

int main(){
    Inventario I1;

    cout << "\n";
    system("color 09");
    cout << "Ingresa el nombre del producto: ";
    getline(cin,I1.nombre);
    cout << "Ingresa el codigo de barras: ";
    getline(cin,I1.codigo);
    cout << "Ingresa la cantidad del producto: ";
    cin >> I1.cantidad;
    cout << "Ingresa el precio del producto: ";
    cin >> I1.costo;
    I1.ganancia= I1.costo * I1.cantidad;
    cout << "\n";
    cout << "Nombre del producto: " << I1.nombre << "\n";
    cout << "Codigo de barras: " << I1.codigo << "\n";
    cout << "Cantidad del producto: " << I1.cantidad << "\n";
    cout << "Precio: $" << I1.costo << "\n";
    cout << "Ganancias del producto: $" << I1.ganancia << "\n";

    cout << "\n \n";
    cout << "------------------------------------------------------------------------------------------\n";
	cout << "Creador: Jesus A. Mex Damian 15/02/2023.";
    return 0;
}

File: 2.POO/Resto/Inventario/Primera version/Java/Inventario.java
import java.util.Scanner;
public class Inventario {
    public static void main(String[] args) {
        Scanner mex= new Scanner(System.in);

        Inventarios I1 = new Inventarios();
        System.out.println("Ingresa el nombre del producto: ");
        I1.nombre= mex.nextLine();
        System.out.println("Ingresa un codigo: ");
        I1.codigo= mex.nextLine();
        System.out.println("Ingresa la cantidad de producto: ");
        I1.cantidad= mex.nextDouble();
        System.out.println("Ingresa el costo del producto: ");
        I1.costo= mex.nextDouble();
        I1.ganancia= I1.cantidad * I1.costo;

        System.out.println("Producto: " + I1.nombre);
        System.out.println("Codigo: " + I1.codigo);
        System.out.println("Cantidad del producto: " + I1.cantidad);
        System.out.println("Precio: " + I1.costo);
        System.out.println("Ganancia total: " + I1.ganancia);
    }
}
